### PR TITLE
feat(issues): Prompt to setup releases in resolve dropdown

### PR DESCRIPTION
**Pull request by scttcper**
Displays a prompt to setup releases in the resolve button "more options" dropdown

image

### CODE REVIEW

**Review by malwilley — APPROVED** [review]

File: static/app/components/actions/resolve.tsx [hunk 1]
```diff
@@ -215,7 +249,13 @@ function ResolveActions({
             ? ['next-release', 'current-release', 'another-release']
             : []
         }
-        menuTitle={t('Resolved In')}
+        menuTitle={
```
Comment (malwilley) [inline static/app/components/actions/resolve.tsx:252]: Seems a bit hacky but idk if there's a better way

Comment (scttcper) [reply static/app/components/actions/resolve.tsx:252]: yeah not my favorite but it does work really well

Comment (scttcper) [reply static/app/components/actions/resolve.tsx:252]: items is much more difficult to use

Comment (malwilley) [reply static/app/components/actions/resolve.tsx:252]: I guess the alternative is not using the dropdown menu at all. You could use DropdownTrigger and `useOverlay` for this case. Probably not be worth the effort if this works well

**Pull request merged**